lib/hydra/works/models/concerns/generic_work_behavior.rb

```diff
@@ -20,11 +20,48 @@ module GenericWorkBehavior
       type [RDFVocabularies::PCDMTerms.Object,WorksVocabularies::WorksTerms.GenericWork]
       include Hydra::Works::AggregatesGenericFiles
       include Hydra::Works::AggregatesGenericWorks
+      include Hydra::Works::BlockChildObjects
+
+      filters_association :members, as: :child_generic_works, condition: :works_generic_work?
```

Comment:
Okay. I thought GenericWork was a restricted case where it didn't have work parents. (@elrayle yes, I meant PCDM::Object)